Makefile

```diff
@@ -86,3 +86,6 @@ dist: clean ## builds source and wheel package
 
 install: clean ## install the package to the active Python's site-packages
 	pip install .
+
+minimal-ee:
+	ansible-builder build -f ./minimal-decision-environment.yml -t minimal-de:latest
```

Comment:
should we add ansible-builder to the dev requirements?

Reply:
I'll add that.